components/family-studies/ui/src/main/resources/PhenoTips/FamilySelector.xml

```diff
@@ -129,8 +129,7 @@
 
             var _this = this;
 
-            this.familySearch = new XWiki.Document('FamilySearch', 'PhenoTips');
-            this.familySearchUrl = this.familySearch.getURL('get', 'outputSyntax=plain&amp;rand='+ Math.random());
+            this.familySearchUrl = XWiki.contextPath + "/rest/families/suggest";
```

Comment:
Still doesn't work, append `?` at the end: `"/rest/families/suggest?"`

Reply:
@sdumitriu indeed, sorry, URL corrected in next commit